common/src/main/java/no/difi/meldingsutveksling/config/IntegrasjonspunktPropertiesValidator.java

```diff
@@ -6,6 +6,8 @@
 
 public class IntegrasjonspunktPropertiesValidator implements Validator {
 
+    private static String EMPTY_FIELD = "empty_field";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this field "EMPTY_FIELD" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS3008)